cspt/config.py

```diff
@@ -2,11 +2,11 @@
 from datetime import date as dt
 from datetime import timedelta
 # UPDATE: update this each semester
-REPO = 'http://compsys-progtools.github.io/fall024/'
-BASE_URL = 'https://raw.githubusercontent.com/compsys-progtools/fall2024/main/_'
+REPO = 'http://compsys-progtools.github.io/spring2025/'
+BASE_URL = 'https://raw.githubusercontent.com/compsys-progtools/spring2025/main/_'
 
 GH_APPROVERS = ['brownsarahm','marcinpawlukiewicz','VioletVex']
```

Comment:
this shoudl be you & your TAs now
```suggestion
GH_APPROVERS = ['AymanBx','trevmoy','efsa95']
```